src/Idris/IBC.hs

```diff
@@ -394,11 +394,17 @@ process reexp phase archive fn = do
                 processAccess reexp phase archive
 
 timestampOlder :: FilePath -> FilePath -> Idris ()
-timestampOlder src ibc = do srct <- runIO $ getModificationTime src
-                            ibct <- runIO $ getModificationTime ibc
-                            if (srct > ibct)
-                               then ifail $ "Needs reloading " ++ show (srct, ibct)
-                               else return ()
+timestampOlder src ibc = do
+  srct <- runIO $ getModificationTime src
+  ibct <- runIO $ getModificationTime ibc
+  if (srct > ibct)
+    then ifail $ unlines [ "Module needs reloading:"
+                         , unwords ["\tSRC :", show src]
+                         , unwords ["\tModified at:", show srct]
+                         , unwords ["\tIBC :", show src]
+                         , unwords ["\tModified at:", show srct]
```

Comment:
The IBC lines also refer to `src` and `srct`, and `ibc` and `ibct` are not output. Is this a copypaste error?